README.md

```diff
@@ -20,21 +20,23 @@ For extended documentation see [here](https://developer.here.com/rest-apis/docum
 
 ### Installation
 
-Make sure you have already installed the latest version of Node (at least version 4.2) and NPM (Node Package Manager) and then run:
-
-    npm install
+Make sure you have already installed the latest version of `Node` (at least version 6.10.3) and `NPM` (Node Package Manager) and then run:
```

Comment:
Strictly speaking, I would not use `code` formatting here, as you're referring to the literal name of the apps, not to the name of the command line executables. So, IMO, either use Node and NPM, or `node` and `npm` (lower case).